src/ResourceManager/SubscriptionDefinition/Commands.SubscriptionDefinition.Test/ScenarioTests/SubscriptionDefinitionTests.ps1

```diff
@@ -0,0 +1,60 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+List subscription definitions
+#>
+function Test-ListSubscriptionDefinitions
+{
+    $definitions = Get-AzureRmSubscriptionDefinition
+
+    Assert-AreEqual 4 $definitions.Count
+	Foreach($def in $definitions)
+	{
+		Assert-NotNull $def.Name
```

Comment:
Should do more complete assertions over expected properties

Reply:
Done